homeassistant/components/pvoutput/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "To authenticate with PVOutput you'll need to get the API key at https://pvoutput.org/account.jsp.\n\nThe system IDs of registered systems are listed on that same page.",
```

Comment:
Maybe use a description placeholder for the url?

Reply:
Done